zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -11,17 +11,17 @@ externalDocs:
   url: https://docs.camunda.io/docs/apis-tools/zeebe-api-rest/overview/
 
 servers:
-  - url: "{schema}://{host}:{port}/v1"
+  - url: "{schema}://{host}:{port}/v2"
```

Comment:
:question: Should we create a new file now for `v2` of the OpenAPI spec? How will users that wish to continue to use the `v1` API and URLs be able to generate clients for it?

Reply:
Yep, good catch. I'll add another YAML file for v1 👍 
That way, users can generate a specific v1 client and we can (hopefully) introduce a version dropdown easily in our public docs as well.